chat-plugins/shop.js

```diff
@@ -348,73 +348,31 @@ exports.commands = {
 	atm: function(target, room, user) {
 		if (!this.canBroadcast()) return;
 		if (!target) target = user.name;
-		var originalName = Tools.escapeHTML(target);
-		target = toId(target);
-		function atm(target) {
-			var bucks = economy.readMoney(target);
-			var label = (bucks == 1 ? ' Gold buck' : ' Gold bucks');
-			var output = "<u>Gold Wallet:</u><br />";
-			switch (bucks) {
-				case 0:
-					output += "<b><font color=\"" + Gold.hashColor(target) + "\">" + originalName + "</font></b> does not have any Gold bucks.";
-					break;
-				default:
-					output += "<b><font color=\"" + Gold.hashColor(target) + "\">" + originalName + "</font></b> has " + bucks + label + ".";
-			}
-			return output;
-		}
-		return this.sendReplyBox(atm(target));
+		target = (Users.getExact(target) ? Users.getExact(target).name : target);
+		var bucks = economy.readMoney(toId(target)) || 'does not have any';
+		if (parseInt(bucks)) bucks = 'has ' + bucks;
+		var output = '<u>Gold Wallet:</u><br />' + 
+			'<b><span style = "color:' + Gold.hashColor(toId(target)) + '">' + Tools.escapeHTML(target) + '</span></b> ' + bucks + ' Gold ' +
```

Comment:
``Gold.hashColor();`` automatically takes the id, this ``toId();`` is not needed - please remove it